schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
+					"items": {
+						"type": "string"
+					},
+					"minItems": 1
+				},
+				"gazetteerEntries": {
+					"title": "Gazetteer entries",
+					"type": "array",
+					"description": "Entries from geographical indices or directories describing the geographical area. This field should be used to describe subnational coverage. Use of is recommended.",
+					"items": {
+            "$ref": "#/$defs/Gazetteer_entry"
+          },
+          "uniqueItems": true,
+					"minItems": 1
+				},
+				"bbox": {
+					"title": "Bounding box",
+					"type": "array",
+					"description": "A geographic bounding box delimiting the geographical area.",
+					"items": {
+						"type": "number"
+					},
+					"minItems": 1
```

Per https://datatracker.ietf.org/doc/html/rfc7946#section-5

```suggestion
					"minItems": 4,
					"maxItems": 4
```